recipes/pyotp/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pyotp" %}
+{% set version = "2.2.6" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "dd9130dd91a0340d89a0f06f887dbd76dd07fb95a8886dc4bc401239f2eebd69" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install --no-deps . -vv
+  noarch: python
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - pyotp
+
+about:
+  home: https://github.com/pyotp/pyotp
+  license: BSD
```

Comment:
This should state [which variant of the BSD licence it is](https://opensource.org/licenses/alphabetical) like `BSD-2-Clause` or `BSD-3-Clause`.

Reply:
It was really an MIT license according to https://pypi.org/project/pyotp/